src/views/incident/IncidentView.test.tsx

```diff
@@ -832,6 +872,19 @@ describe('Incidents Table: reflects incident modifications', () => {
             display: "REO"
           },
           description: ""
+        })
+        .onGet("/api/")
+        .reply(200, {
+          "server-version": "1.2.3",
+          "api-version": {
+            "stable": "v1",
+            "unstable": "v2"
+          },
+          "jsonapi-schema": {
+            "stable": "/api/schema/",
+            "v1": "/api/v1/schema/",
+            "v2": "/api/v2/schema/"
+          }
```

Comment:
Would it be possible to move the inner object mocking the reply from the server to a named constant so that there is only one place to update when/if the API changes?

This bit:

```json
{
  "server-version": "1.2.3",
  "api-version": {
     "stable": "v1",
     "unstable": "v2"
  },
  "jsonapi-schema": {
     "stable": "/api/schema/",
     "v1": "/api/v1/schema/",
     "v2": "/api/v2/schema/"
  }
}
```

Reply:
Will do! This one was written in a hurry. 😅